wsgi.py

```diff
@@ -7,367 +7,336 @@
 from flask_httpauth import HTTPBasicAuth
 import moc_openshift
 
-APP = Flask(__name__)
-AUTH = HTTPBasicAuth()
-
 OPENSHIFT_URL = os.environ["OPENSHIFT_URL"]
 ADMIN_USERNAME = os.environ.get("ACCT_MGT_ADMIN_USERNAME", "admin")
 ADMIN_PASSWORD = os.environ["ACCT_MGT_ADMIN_PASSWORD"]
 AUTH_TOKEN = os.environ.get("ACCT_MGT_AUTH_TOKEN")
 
-if __name__ != "__main__":
-    APP.logger = logging.getLogger("gunicorn.error")
-    # logger level INFO = 20 see (https://docs.python.org/3/library/logging.html#levels)
-    APP.logger.setLevel(20)
-
-
-class MocOpenShiftSingleton:
-    """Ensure a single instance of the OpenShift API"""
 
-    # pylint: disable=too-few-public-methods
-    class MocOSInt:
-        """Wrapper for API openshift API class"""
+def get_openshift(url, token, logger):
+    if token is None:
+        with open("/var/run/secrets/kubernetes.io/serviceaccount/token", "r") as file:
+            token = file.read()
 
-        def __init__(self, url, logger):
-            token = AUTH_TOKEN
-            if token is None:
-                with open(
-                    "/var/run/secrets/kubernetes.io/serviceaccount/token", "r"
-                ) as file:
-                    token = file.read()
-            self.shift = moc_openshift.MocOpenShift4x(url, token, logger)
-            APP.logger.info("using Openshift ver 4")
+    return moc_openshift.MocOpenShift4x(url, token, logger)
```

Comment:
doesn't this make the garbage collection work harder, however, I am going to let this one pass, because this project absolutely need to move forward.